src/shiftedNormL1.jl

```diff
@@ -67,8 +67,12 @@ function iprox!(
   @. y = -ψ.xk - ψ.sj
 
   for i ∈ eachindex(y)
-    @assert d[i] > 0
-    y[i] = min(max(y[i], -g[i] / d[i] - λ / d[i]), -g[i] / d[i] + λ / d[i])
+    if d[i] < 0
```

Comment:
Is it < or <= ?

Reply:
You're right; it's '<'. We need to discuss the sign of the affine part if $d[I]=0$.